docs/ref/models/querysets.txt

```diff
@@ -1843,6 +1843,16 @@ Example::
 
 If you pass ``in_bulk()`` an empty list, you'll get an empty dictionary.
 
+  .. versionchanged:: 1.10
```

Comment:
Try to make the annotation "self-contained" as described in https://docs.djangoproject.com/en/dev/internals/contributing/writing-documentation/#documenting-new-features.  This line shouldn't be indented either.